zephyr/module.yml

```diff
@@ -0,0 +1,5 @@
+---
+name: zephyr-config
```

Comment:
This should be something better matching, e.g. 


```suggestion
name: zaphod-config
```

Reply:
Oh my goodness! I must have gone a little snow-blind with all the Zs and PHs :)